src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int BALL_COUNT = 100;
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
+        ball.setNumber(new Random().nextInt(BALL_COUNT));
```

Comment:
Don't use static methods in your solution. While there is no static method here, the usage of `new Random()` could lead to less efficient code if called multiple times. Consider making `Random` a private instance variable.